projects/plugins/jetpack/modules/videopress/class.videopress-gutenberg.php

```diff
@@ -191,6 +192,24 @@ public function render_video_block_with_videopress( $attributes, $content ) {
 		);
 	}
 
+	/**
+	 * Temporary method to enable resumable uploads for testing by Automatticians
+	 */
+	public function add_resumable_upload_support() {
+		if (
+			defined( 'IS_WPCOM' ) && IS_WPCOM &&
+			function_exists( 'is_automattician') && is_automattician()
+		) {
+			wp_enqueue_script(
+				'videopress-add-resumable-upload-support',
+				plugins_url( 'js/videopress-add-resumable-upload-support.js', __FILE__ ),
+				null,
+				'1',
+				false
+			);
+		}
```

Comment:
@pgk you can add this block of code here to enable resumable uploads on Jetpack for testing:

`		if ( function_exists( 'is_automattician' ) && is_automattician() ) {
			wp_add_inline_script(
				'jetpack-videopress-gutenberg-override-video-upload',
				'var videoPressResumableEnabled = true;'
			);
		};
`